templates/pioneer-citp/news-disinfo-study/news-disinfo-study.1.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+    "mozPipelineMetadata": {
+        "bq_metadata_format": "pioneer"
+    },
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object", "properties": {
+        "WebScience.Measurements.PageNavigation": { "type": "object", "properties": {
+            "numUntrackedVisits": {"type": "number"},
```

Comment:
`number` is treated as FLOAT64, you can also specify `integer` for an INT64.

Reply:
:+1: